barretenberg/cpp/src/barretenberg/ecc/fields/field_conversion.test.cpp

```diff
@@ -34,6 +34,9 @@ TEST_F(FieldConversionTest, FieldConversionFr)
 
     bb::fr x2(bb::fr::modulus_minus_two); // modulus - 2
     check_conversion(x2);
+
+    bb::fr x3(curve::Grumpkin::Group::affine_point_at_infinity.x);
```

Comment:
Somehow, this test passes. I guess, its somewhat random if a value gets mapped back to itself if we call from_montgomery_form() and to_montgomery_form().